components/brave_shields/adblock/rs/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "adblock-cxx"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+cxx = "1.0"
+adblock = { git = "https://github.com/djandries/adblock-rust", default-features = false, features = ["full-regex-handling", "object-pooling", "unsync-regex-caching", "debug-info", "css-validation"] }
```

Comment:
Note that as of https://github.com/brave/brave-core/pull/17629 this should now be `v0.7.7`